_data/simple-icons.json

```
@@ -16264,6 +16264,11 @@
 		"hex": "BE3939",
 		"source": "https://www.spriters-resource.com"
 	},
+	{
+		"title": "The Verge",
+		"hex": "5100FF",
```

```suggestion
		"hex": "5200FF",
```

I'm seeing `#5200ff` in their CSS.